examples/TFTGizmo/TFTGizmo.ino

```diff
@@ -25,17 +25,25 @@
   Adafruit_FlashTransport_QSPI flashTransport(PIN_QSPI_SCK, PIN_QSPI_CS,
     PIN_QSPI_IO0, PIN_QSPI_IO1, PIN_QSPI_IO2, PIN_QSPI_IO3);
 #else
-  #if (SPI_INTERFACES_COUNT == 1)
+  #if (SPI_INTERFACES_COUNT == 1 || defined(ADAFRUIT_CIRCUITPLAYGROUND_M0))
     Adafruit_FlashTransport_SPI flashTransport(SS, &SPI);
   #else
     Adafruit_FlashTransport_SPI flashTransport(SS1, &SPI1);
   #endif
 #endif
+
+// TFT SPI interface selection
+#if (SPI_INTERFACES_COUNT == 1)
```

Comment:
|| defined(ADAFRUIT_CIRCUITPLAYGROUND_M0))

Reply:
(right?)

Reply:
No, for TFT SPI, it's SPI1 on the CPX.

Reply:
I forgot to mention, that I tested it. :)

Reply:
> oh right ya! ok